scss/partials/_columns.scss

```diff
@@ -5,7 +5,30 @@
   padding-right: 0;
 
   &--post-hero {
-    width: calc(min(62.5rem, 100%));
+    padding-top: 9.6rem;
+    
+    @include breakpoint(desktop) {
+      grid-template-columns: calc(min(62.5rem, 50%)) auto;
+      margin-bottom: 9.4rem;
+      margin-left: auto;
+      margin-right: auto;
+      max-width: 144rem;
+      padding-top: 27.4rem;
+    }
+
+    @include grid();
+
+    p:not(:last-child) {
+      margin-bottom: 3rem;
+    }
+
+    // Special case for section heading until variablized for a modifier in Velocity
+    @at-root #{&} {
+      h2.section-heading {
```

Comment:
Ultimately, this should end up being a modifier on the section-heading block, but it needs VTL support to pull off, and I didn't want to go that far with the refactor now. This isn't ideal though and should be considered for refactoring later.